src/gennylib/src/topology.cpp

```diff
@@ -177,7 +177,7 @@ void Topology::computeDataMemberConnectionStrings(DBConnection& connection) {
         bsoncxx::array::view passives_view = passives.get_array();
         for (auto member : passives_view) {
             MongodDescription memberDesc;
-            memberDesc.mongodUri = std::string(member.get_utf8().value);
+            memberDesc.mongodUri = nameToUri(std::string(member.get_utf8().value));
```

Comment:
Can you add context about why we need to do this? Is it related to the other parts of this PR?

Reply:
I think it was just a bug in the code, as mongodUri should append required username and password and convert the host name to a host URI to be used for the client exactly as the [previous loop](https://github.com/mongodb/genny/blob/869fe58310e95d39b6e1d1a63b15f0723f8a2d0d/src/gennylib/src/topology.cpp#L162)